python/ray/rllib/common.py

```diff
@@ -30,16 +59,22 @@ class Algorithm(object):
   TODO(ekl): support checkpoint / restore of training state.
   """
 
-  def __init__(self, env_name, config):
+  def __init__(self, env_name, config, s3_bucket=None):
+    self.experiment_id = uuid.uuid4()
```

Comment:
Right now we only dump the config log here and then the rest is dumped in train.py using the S3Logger class; the bucket or handle is not needed here right now, so for now it's fine to not save it I think